lib/components/Input/base/base.scss

```diff
@@ -118,7 +118,7 @@
             margin-left: var(--core-spacing-200);
         }
         &--disabled {
-            opacity: var(--core-opacity-600);
+            color: var(--component-field-label-color-disabled) !important;
```

Comment:
What is the style we are trying to override and we needed to add `important`. Maybe we can handle the class structure to handle this?

Reply:
There's default color assigned to text component that I'm trying to overwrite. It always has a priority over any color I pass. I'm thinking it should be fixed by adding color prop to text component or not enforcing any default color that can't be changed without higher priority.

Reply:
Ok, I noticed there's already color property in the text component, let me try to use that to overwrite it

Reply:
I removed this style